conektaprestashop.php

```diff
@@ -969,6 +919,49 @@ private function _createWebhook()
             Configuration::updateValue('CONEKTA_WEBHOOK_FAILED_URL', $url);
         }
     }
+
+    /**
+     * Display the transaction status (paid or unpaid) and mode
+     *
+     * @return string HTML/JS Content
+     */
+
+    public function getTransactionStatus($id_order)
+    {
+        if (Db::getInstance()->getValue(
```

Comment:
Que opinas de cambiar esta query a un query method para que al final quede algo como:

```
public function module($order_id) {
   return Db::getInstance()->getValue(
            'SELECT module FROM ' . _DB_PREFIX_ . 'orders '
            .'WHERE id_order = ' . pSQL((int) $id_order))
}
```

y el if:

`if(module(order_id) == $this->name)`

igual para la linea 935